ajax/libs/vanilla-lazyload/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "vanilla-lazyload",
+  "filename": "dist/lazyload.min.js",
+  "version": "2.1.2",
+  "description": "LazyLoad is a fast, lightweight and flexible script for loading images only when they're about to enter the viewport of a scrollable area, with an excellent support to the progressive JPEG image format. Typescript module definition is available.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/verlok/lazyload.git"
+  },
+  "license": "MIT",
+  "authors": [
+    "Andrea",
+    "verlok",
+    "Verlicchi"
+  ],
+  "keywords": [
+    "lazy load",
+    "progressive JPEG",
+    "performance",
+    "no-jquery"
+  ],
+  "homepage": "https://verlok.github.io/lazyload",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/verlok/lazyload.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "src",
```

Comment:
@maruilian11 I suggested the config due to the result should look like the following in this case:
```
$ tree 2.1.2/
2.1.2/
├── lazyload.js
├── lazyload.min.js
└── lazyload.min.js.map
```